### PR TITLE
chore(ci): add windows as target build platform

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

**Comment by IceTDrinker** [comment]
c api tests will fail the windows build, it will propbably require disabling them

File: Makefile [hunk 1]
```diff
@@ -226,11 +227,18 @@ format_doc_latex:
 	@printf "\n===============================\n"
 
 .PHONY: check_compile_tests # Build tests in debug without running them
+ifneq ($(filter $(OS),Linux Darwin),)
```
Comment (IceTDrinker) [inline Makefile:230]: looks to be working well but I'm not a fan of condiftional Makefiles

could we do

```
.PHONY: check_compile_tests # Build tests in debug without running them
check_compile_tests: build_c_api
	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) test --no-run \
		--features=$(TARGET_ARCH_FEATURE),experimental,boolean,shortint,integer,internal-keycache \
		-p tfhe && \
		if [[ "$(OS)" == "Linux" || "$(OS)" == "Darwin" ]]; then \
			./scripts/c_api_tests.sh --build-only; \
		fi
```
instead ?

Comment (soonum) [reply Makefile:230]: Yes we can do it at the cost of losing the `&&` logic here.

Comment (IceTDrinker) [reply Makefile]: hmm why do we lose the && logic ? it's still in there

Comment (IceTDrinker) [reply Makefile]: I'm not even sure the && is required in that case anyways

Comment (soonum) [reply Makefile]: Keeping `&& \` as is, is a syntax error.

Comment (soonum) [reply Makefile]: Correction, it's a syntax error when followed by a `@` right before the `if`

Comment (IceTDrinker) [reply Makefile]: anyways as you said let's remove that &&

**Comment by IceTDrinker** [comment]
@slab-ci cpu_test

**Pull request merged**